.storybook/preview.js

```diff
@@ -22,6 +22,44 @@ configureActions({
   limit: 20,
 });
 
+const customViewports = {
+  extraSmall: {
```

Comment:
according to the link you provided in Jira ticket - the values should be applied only for a `width` but not for a `height`

Reply:
The styles object requires a width and a height. I choose the two values for each viewport to be the minimum and maximum width, this way you could switch between them fast using the flip feature in storybook.

Reply:
seems logical :)